sysboundary/ionosphere.cpp

```diff
@@ -75,7 +75,7 @@ namespace SBC {
    Real Ionosphere::couplingTimescale; // Magnetosphere->Ionosphere coupling timescale (seconds)
    Real Ionosphere::couplingInterval; // Ionosphere update interval
    int Ionosphere::solveCount; // Counter of the number of solvings
-   Real Ionosphere::backgroundIonisation; // Background ionisation due to stellar UV and cosmic rays
+   Real Ionosphere::backgroundIonization; // Background ionization due to stellar UV and cosmic rays
```

Comment:
We used to have the policy, inherited from Ilja, to use American spelling cuz it's less typin. Feel free to overhaul that but it can cause headaches later...

Reply:
Nah, I'm fully ok with that y'all.